shopsafe-frontend/src/app/components/feedback-forms/feedback-forms.component.html

```diff
@@ -0,0 +1,33 @@
+<div id="title" class="segment">
+    <div class="segment-text">
+        <div class="heading-text">
+            <h1>Feedback Forms</h1>
+            <p>Help us improve ShopSafe by giving us feedback!</p>
+        </div>
+
+        <div class="section">
+            <h2><i class="fas fa-shopping-cart"></i> User Experience Form</h2>
+            <div class="box">
+                <p>
+                   Let us know what your experience with ShopSafe was like by filling out this
+                   <a href="https://docs.google.com/forms/d/e/1FAIpQLSeagc76SWGCG8LTBVM_ZrWZAjAoGeH2fjI2fWR15o9Z-x7J-Q/viewform?usp=sf_link">
+                    form
+                    </a>. 
+                </p>
+            </div>
+        </div>
+
+        <div class="section">
+            <h2><i class="fas fa-bug"></i> Bug Reporting</h2>
```

Comment:
Maybe you could try including the bug icon in the footer linking to Feedback instead of within the template for styling consistency 0: